src/main/java/seedu/address/model/displayable/Phone.java

```diff
@@ -1,18 +1,24 @@
 package seedu.address.model.displayable;
 
 import static java.util.Objects.requireNonNull;
-import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import seedu.address.commons.util.AppUtil;
+
+import static seedu.address.commons.util.AppUtil.validateArgument;
 
 /**
  * Represents a Displayable's phone number in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidPhone(String)}
  */
 public class Phone {
 
-
-    public static final String MESSAGE_CONSTRAINTS =
-            "Phone numbers should only contain numbers, and it should be at least 3 digits long";
-    public static final String VALIDATION_REGEX = "\\d{3,}";
+    public static final String MESSAGE_CONSTRAINTS = "Phone numbers must contain at least one number.";
+    public static final String MESSAGE_RECOMMENDATIONS =
+            "Phone numbers should only contain numbers, and it should be at least 3 digits long. "
+                    + "Area codes are allowed, signified by a '+' and up to 3 numbers, followed by a space separating "
+                    + "this from the main number.";
+    public static final String VALIDATION_REGEX = ".*\\d.*";
+    public static final String AFFIRMATION_REGEX = "((\\+\\d{0,3} )?)\\d{3,}";
```

Comment:
Appreciate the clear messages explaining the distinction between invalid inputs which throw exceptions, vs valid but unrecommended inputs which add to the list of warnings